samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -106,7 +108,12 @@ static const char link_usage_str[] =
 	"  psm:          Enable/disable Power Saving Mode (PSM) with\n"
 	"                default or with custom parameters\n"
 	"  rai:          Enable/disable RAI feature. Actual use must be set for commands\n"
-	"                supporting RAI. Effective when going to normal mode.\n";
+	"                supporting RAI. Effective when going to normal mode.\n"
+	"  dnsaddr:      Set a secondary DNS address.\n"
```

Comment:
I would emphasize already in a command name that this is for secondary DNS address. Also in LINK_CMD_DNSADDR that could be LINK_CMD_SEC_DNSADDR for example.

Reply:
I tried using the same command name as the nrf function name (`link setdnsaddr`), but this didn't look good because it can both read and write. Something like `link secdnsaddr` looks like it's security related, not secondary.

Any good other suggestions?

Reply:
Good point. Naming is always hard. Somehow, I would emphasize still that isn't a primary and actually not a secondary either, it is a backup/fallback DNS. So, would backup_dnsaddr or fallback_dnsaddr.

Reply:
I also think for example "backup" would be a better term, so that it wouldn't get confused with the DNS addresses received from the network. And it would be good to also mention this in the command help text.

Reply:
Documentation is updated.